codex-rs/core/src/mcp_connection_manager.rs

```diff
@@ -208,7 +211,7 @@ impl McpConnectionManager {
             ) && !use_rmcp_client
             {
                 info!(
-                    "skipping MCP server `{}` configured with url because rmcp client is disabled",
+                    "skipping MCP server `{}` because the legacy MCP client only supports stdio servers",
```

Comment:
```suggestion
                    "skipping MCP server `{server_name}` because the legacy MCP client only supports stdio servers",
```

I guess Clippy knows about `format!()` but not our tracing macros.